src/arch/arm/regs/misc.cc

```diff
@@ -1928,10 +1928,18 @@ faultSctlr2EL1(const MiscRegLUTEntry &entry,
     if (HaveExt(tc, ArmExtension::FEAT_SCTLR2)) {
         const SCR scr = tc->readMiscReg(MISCREG_SCR_EL3);
         const HCRX hcrx = tc->readMiscReg(MISCREG_HCRX_EL2);
-        if (auto fault = faultHcrFgtEL1<read, g_bitfield, &HFGTR::sctlrEL1>(entry, tc, inst);
-            fault != NoFault) {
+        if (
```

Comment:
I'm not sure what these ARM changes have to do with the LULESH benchmark?

Reply:
I'm going to separate this out, but TL;DR I needed to add this here for now to commit. "misc.cc" breaks the line-length rules and my pre-commit was complaining. I'll let you know when it's gone

Reply:
I think I have seen this when merging develop into my private branch. Thanks for fixing this!